superset-frontend/src/visualizations/presets/MainPreset.js

```diff
@@ -122,7 +123,6 @@ export default class MainPreset extends Preset {
         new TableChartPlugin().configure({ key: 'table' }),
         new TimePivotChartPlugin().configure({ key: 'time_pivot' }),
         new TimeTableChartPlugin().configure({ key: 'time_table' }),
-        new TreemapChartPlugin().configure({ key: 'treemap' }),
```

Comment:
Maybe not a direct replacement from nvd3-treemap to echart -treemap
1) We should need some migration scripts,  adapt to the new Treemap.
2) fix original cypress CI